requirements-dev.txt

```diff
@@ -8,8 +8,11 @@ mock==1.3.0
 h5py==2.6.0
 nose==1.3.7
 numpy==1.11.1
-psutil==3.4.2
 scipy==0.17.1
+psutil==3.4.2
 shapely==1.5.13
 docutils==0.12
 decorator==4.0.6
+funcsigs==1.0.2
+pbr==1.8.0
+six==1.10.0
```

Comment:
Those are dependencies of our dependencies. I've simply added them as explicit dependencies to have -dev and -py27/35 aligned. I need those in -py27/35 to have them in the mirror.